aspnetcore/includes/benefits.md

```diff
@@ -11,5 +11,5 @@ ASP.NET Core provides the following benefits:
 * Built-in [dependency injection](xref:fundamentals/dependency-injection).
 * A lightweight, [high-performance](https://github.com/aspnet/benchmarks), and modular HTTP request pipeline.
 * Ability to host on [IIS](xref:host-and-deploy/iis/index), [Nginx](xref:host-and-deploy/linux-nginx), [Apache](xref:host-and-deploy/linux-apache), [Docker](xref:host-and-deploy/docker/index), or self-host in your own process.
-* Side-by-side app versioning when targeting [.NET Core](/dotnet/articles/standard/choosing-core-framework-server).
+* Side-by-side app versioning when targeting [.NET Core](/dotnet/standard/choosing-core-framework-server#a-need-for-side-by-side-of-net-versions-per-application-level).
```

Comment:
The link text ".NET Core" is misleading now for the target.  I would put this link on the phrase "side-by-side app versioning" and you could remove the link from ".NET Core"

Reply:
Submitted a correction for this, thanks Tom!